tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -643,6 +653,7 @@ public void ttlRename() throws Exception {
 
     @Override
     public Void call() throws Exception {
+      AuthenticatedClientUser.set("test");
```

Comment:
Yes, because `AuthenticatedClientUser` is thread-local. For concurrent create tests, we have to make sure each thread has the user setup.